app/components/widgets/CustomTooltip.scss

```diff
@@ -13,6 +13,18 @@
       margin-left: 20px; // offset = horizontal padding of icon and tooltip
     }
   }
+
+  .PoolAvatarTooltip {
```

Comment:
Usually we deal with this with `:global` overrides inside the files that need it, but this is also fine I think (since the overrides also tend to be messier to work with, I wouldn't really call them a better solution in any way)

Reply:
I was thinking to have more options this way, instead of having `isAligningRight` or `SimpleCenteredTooltip` as fixed conditional class